GitCommands/Logging/CommandLog.cs

```diff
@@ -55,22 +67,53 @@ internal CommandLogEntry(string fileName, string arguments, DateTime startedAt,
             IsOnMainThread = isOnMainThread;
         }
 
-        public override string ToString()
+        public string ColumnLine
         {
-            var duration = Duration == null
-                ? "running"
-                : $"{((TimeSpan)Duration).TotalMilliseconds:0}ms";
+            get
+            {
+                var duration = Duration == null
+                    ? "running"
+                    : $"{((TimeSpan)Duration).TotalMilliseconds:0}ms";
 
-            var fileName = FileName;
+                var fileName = FileName;
 
-            if (fileName.EndsWith("git.exe"))
-            {
-                fileName = "git";
-            }
+                if (fileName.EndsWith("git.exe"))
+                {
+                    fileName = "git";
+                }
+
+                var pid = ProcessId == null ? "     " : $"{ProcessId,5}";
+                var exit = ExitCode == null ? "  " : $"{ExitCode,2}";
 
-            var pid = ProcessId == null ? "     " : $"{ProcessId,5}";
+                return $"{StartedAt:HH:mm:ss.fff} {duration,7} {pid} {(IsOnMainThread ? "UI" : "  ")} {exit} {fileName} {Arguments}";
+            }
+        }
 
-            return $"{StartedAt:HH:mm:ss.fff} {duration,7} {pid} {(IsOnMainThread ? "UI" : "  ")} {fileName} {Arguments}";
+        public string Detail
+        {
+            get
+            {
+                var s = new StringBuilder();
+
+                s.Append("File name:  ").AppendLine(FileName);
+                s.Append("Arguments:  ").AppendLine(Arguments);
```

Comment:
Hey! Now I have to type git manually, cannot copy the complete command in one go!

Reply:
Ahh. The intent here was to show the full path for the executable somewhere.

Would a context menu "Copy" from the list box, with a Ctrl+C shortcut, work?

Reply:
I can live with typing "git" (or copying the file) when needed. 
A generic context menu with Copy would be an improvement, now you have to try and see if Ctrl-C works . Not important but helps the user, it is not ideal to view the stacktrace in the scrollbox. (I just want to view the stacktrace in a text editor, I do not want more support here!)